src/codegen/riscv64/macro-assembler-riscv64.cc

```diff
@@ -3645,17 +3645,20 @@ void TurboAssembler::MulOverflow(Register dst, Register left,
          overflow != scratch);
   DCHECK(overflow != left && overflow != right_reg);
 
+
```

Comment:
Please use `clang-format` to keep consistent formatting. This extra empty line should be removed.

Reply:
Thank you, I will fix it.